flood_adapt/object_model/hazard/hazard.py

```diff
@@ -44,11 +48,32 @@ def __init__(self, scenario: ScenarioModel, database_input_path: Path) -> None:
         self.site = Site.load_file(
             database_input_path.parent / "static" / "site" / "site.toml"
         )
+
         self.set_simulation_paths()
+
+        self.set_sfincs_map_path(mode=self.event_mode)
+
         self.has_run = self.sfincs_has_run_check()
 
+    @property
+    def event_mode(self) -> Mode:
+        return self._mode
+
+    @event_mode.setter
+    def event_mode(self, mode: Mode) -> None:
+        self._mode = mode
+
+    def set_sfincs_map_path(self, mode: Mode) -> None:
```

Comment:
you mean  that I should explicitly use set_.. and get_..?
With python you can define getter and setter via decorators @property and @...setter. It is basically the same as
get_event_mode and set_event_mode.

Reply:
No I mean if you use a function only within the class, you can mark it by using _set_sfincs_map_path. That specifies that other people using the class shouldn't use this function since it is only there for use within the function

Reply:
ohhwww yeah correct!